src/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Digest.cs

```diff
@@ -16,27 +16,17 @@ internal static partial class AppleCrypto
         [DllImport(Libraries.AppleCryptoNative, EntryPoint = "AppleCryptoNative_DigestCreate")]
         internal static extern SafeDigestCtxHandle DigestCreate(PAL_HashAlgorithm algorithm, out int cbDigest);
 
-        internal static unsafe int DigestUpdate(SafeDigestCtxHandle ctx, ReadOnlySpan<byte> pbData, int cbData)
-        {
-            fixed (byte* pbDataPtr = &pbData.DangerousGetPinnableReference())
-            {
-                return DigestUpdate(ctx, pbDataPtr, cbData);
-            }
-        }
+        internal static int DigestUpdate(SafeDigestCtxHandle ctx, ReadOnlySpan<byte> pbData, int cbData) =>
+            DigestUpdate(ctx, ref pbData.DangerousGetPinnableReference(), cbData);
```

Comment:
I'm pretty sure there's inconsistent style to that point; but there should be some precedent.  Early on I was exposing directly what the shim needed, then I started doing wrappers like that for legibility, convenience, and safety.